sycl/test-e2e/ESIMD/printf.cpp

```diff
@@ -14,8 +14,8 @@
 // Test using __SYCL_USE_NON_VARIADIC_SPIRV_OCL_PRINTF__ is disabled until ESIMD
 // supports it.
 // FIXME: enable that test
```

Comment:
@turinevgeny, please, update the comments too.

Reply:
Yeah, thanks, missed it.